Add documents for `information_schema.tidb_index_usage` and `sys.schema_unused_index`. (#16446)
pingcap/docs: sys-schema.md

---
title: sys Schema
summary: Learn about the system tables in the `sys` schema.
---

# `sys` Schema

> TiDB provides `sys` Schema since v8.0.0. You can use the views in `sys` schema to understand the data in TiDB's system tables, `INFORMATION_SCHEMA` database, and `PERFORMANCE_SCHEMA` database.

Reviewer: ```suggestion
Starting from v8.0.0, TiDB provides the `sys` Schema. You can use the views in `sys` schema to understand the data in the system tables, [`INFORMATION_SCHEMA`](/information-schema/information-schema.md), and [`PERFORMANCE SCHEMA`](/performance-schema/performance-schema.md) of TiDB.
```

> The cluster upgrades from version prior than v8.0.0 will not have `sys` Schema and the views in it. `sys` and the view inside can be created by the following SQLs:

Reviewer: ```suggestion
## Manually create the `sys` schema and views

For clusters upgraded from versions earlier than v8.0.0, the `sys` schema and the views in it are not created automatically. You can manually create them using the following SQL statements:
```

```sql
CREATE DATABASE IF NOT EXISTS sys;
CREATE OR REPLACE VIEW sys.schema_unused_indexes AS
  SELECT
    table_schema as object_schema,
    table_name as object_name,
    index_name
  FROM information_schema.cluster_tidb_index_usage
  WHERE
    table_schema not in ('sys', 'mysql', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA') and
    index_name != 'PRIMARY'
  GROUP BY table_schema, table_name, index_name
  HAVING
    sum(last_access_time) is null;
```

## `schema_unused_index`

> `schema_unused_index` is used to record indexes that have not been used since the last restart of TiDB. It includes the following columns:

Reviewer: ```suggestion
`schema_unused_index` records indexes that have not been used since the last start of TiDB. It includes the following columns:
```

---
title: sys Schema
summary: Learn about the system tables in the `sys` schema.
---

> # `sys` Schema
> 
> Starting from v8.0.0, TiDB provides the `sys` Schema. You can use the views in `sys` schema to understand the data in the system tables, [`INFORMATION_SCHEMA`](/information-schema/information-schema.md), and [`PERFORMANCE SCHEMA`](/performance-schema/performance-schema.md) of TiDB.

Reviewer 2: ```suggestion
# `sys` schema

Starting from v8.0.0, TiDB provides the `sys` schema. You can use the views in `sys` schema to understand the data in the system tables, [`INFORMATION_SCHEMA`](/information-schema/information-schema.md), and [`PERFORMANCE SCHEMA`](/performance-schema/performance-schema.md) of TiDB.
```
